benchmark/propagator.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const benchmark = require('./benchmark');
-const opentelemetry = require('@opentelemetry/core');
+const opentelemetry = require('../packages/opentelemetry-core');
```

Comment:
Loading by filepath removes dependency management issues and ensures you're testing the current checked out branch